src/components/cc-addon-features/cc-addon-features.js

```diff
@@ -91,8 +104,22 @@ export class CcAddonFeatures extends LitElement {
     return rawValue;
   };
 
-  // Here we sort feature by name (lower case) but first we force a specific order with SORT_FEATURES
+  /**
+   * @param {AddonFeaturesState} state
+   * @returns {state is (AddonFeaturesStateLoading|AddonFeaturesStateLoaded)}
+   * @private
+   */
+  _isLoadedOrLoading (state) {
+    return state.type === 'loaded' || state.type === 'loading';
+  }
+
+  /**
+   * @param {AddonFeature[]} features
+   * @returns {AddonFeatureWithIcon[]}
+   * @private
+   */
   _sortFeatures (features) {
+    // Here we sort feature by name (lower case) but first we force a specific order with SORT_FEATURES
```

Comment:
suggestion: we could move the `Here we sort feature by name (lower case)` part in the JSDoc to know how it will be sorted directly without checking the code of the function.